hand_eye_calibration/NumberedRequest request
---
hand_eye_calibration/CameraPoseDescription description